src/cmd/team.go

```diff
@@ -182,7 +186,7 @@ func GetTeam(key string) (*opslevel.Team, error) {
 
 var listTeamCmd = &cobra.Command{
 	Use:     "team",
-	Aliases: []string{"teams"},
+	Aliases: common.GetAliases("Team"),
```

Comment:
Here specifically, it seems like it makes more sense to leave `Aliases: []string{"teams"},` as is